client/dashboard/sites/settings-site-visibility/style.scss

```diff
@@ -0,0 +1,16 @@
+.dashboard-site-settings-privacy-form .components-checkbox-control:has(input[type="checkbox"]:disabled:checked) {
```

Comment:
This `:has` is not an efficient way to write selectors, but this CSS override doesn't have access to the component code, which would be able to do something more efficient. This is only a fix up for this view until the `CheckboxControl` itself supports rendering a disabled & checked variant of the control.